openshift/e2e-common.sh

```diff
@@ -166,7 +167,11 @@ function deploy_serverless_operator(){
   operator_ns=$(kubectl get og --all-namespaces | grep global-operators | awk '{print $1}')
 
   # Create configmap to use the latest manifest.
-  oc create configmap ko-data -n $operator_ns --from-file="openshift/release/knative-serving-ci.yaml"
+  oc create configmap ko-data-serving -n $operator_ns --from-file="openshift/release/knative-serving-ci.yaml"
+
+  # Create eventing manifest. We don't want to do this, but upstream designed that knative-eventing dir is mandatory
+  # when KO_DATA_PATH was overwritten.
+  oc create configmap ko-data-eventing -n $operator_ns --from-file="openshift/release/knative-eventing-ci.yaml"
```

Comment:
Sorry for hijacking the PR, but I have one general question, perhaps for @mgencur and @lberk 

In openshift/knative-eventing we "patch" the 1.x operator to just intall serving. we basically do `oc apply for our "knative eventing" yamls.

Should we change to this design? But than I am worried that we need to "vendor" some "serving ci yamls"...  But using the `ko-data-eventing` CM would allow us, to use the operator to install "patched" yamls, since for our openshift deliverables we have some little small _desired_ DIFF to pure upstream

Reply:
Yeah. I like this approach from @nak3 because it uses Serverless operator and also the Knative operator to install all the stuff (by feeding the custom CI configs to the operator). This is close to what we do in openshift-knative/serverless-operator because this approach is using operators to install everything. The ways of installing stuff in openshift-knative/serverless-operator and openshift/knative-serving (and eventing) would be so similar that it'd make it mostly redundant to run the same E2E test suites for these repos (IMO).